### ISSUE TITLE
[Bug] MixerState.Transition2D auto-threshold from anguler speed

### ISSUE THREAD

**Issue by ted-hou** [KybernetikGames/animancer#29]
Just a minor issue:

In the inspector `MixerState.Transition2D` the "set threshold from angular speed (deg)" button uses angular speed in radians, while the "set threshold from angular speed (rad)" button seems to be using angular speed in radians multiplied by `pi/180`.

**Comment by KybernetikGames** [comment]
That's a simple fix if you have Animancer Pro:

In MixerState.cs line 820:

```
AddCalculateThresholdsFunctionPerAxis(menu, prefix, "From Angular Speed (Rad)",
    (clip, threshold) => clip.averageAngularSpeed);
AddCalculateThresholdsFunctionPerAxis(menu, prefix, "From Angular Speed (Deg)",
    (clip, threshold) => clip.averageAngularSpeed * Mathf.Rad2Deg);
```

And in LinearMixerState.cs line 301:

```
AddCalculateThresholdsFunction(menu, prefix + "From Angular Speed (Rad)",
    (clip, threshold) => clip.averageAngularSpeed);
AddCalculateThresholdsFunction(menu, prefix + "From Angular Speed (Deg)",
    (clip, threshold) => clip.averageAngularSpeed * Mathf.Rad2Deg);
```

The units being used is the sort of thing you would expect to find in the documentation, but the documentation for the [Motion](https://docs.unity3d.com/ScriptReference/Motion.html) class (the base of `AnimationClip` which contains the `averageAngularSpeed` property) doesn't even have any of its members listed at all (despite the fact that I reported the issue years ago).

**Comment by KybernetikGames** [comment]
This is now fixed in [Animancer v4.3](https://kybernetik.com.au/animancer/docs/changes/animancer-v4-3).
